spec/data/spec.yamlld

```diff
@@ -4,29 +4,46 @@ schema:author:
   - orcid:0009-0001-8740-4213
   - orcid:0000-0001-7508-7428
   - orcid:0000-0001-7046-4474
+
   - $id: https://greggkellogg.net/foaf#me
     dcterms:isReferencedBy: https://greggkellogg.net/foaf.rdf
-  - https://github.com/ioggstream
-  - https://github.com/TallTed
-  - https://github.com/BigBlueHat
+    schema:givenName: Gregg
+    schema:familyName: Kellogg
+
+  - $id: https://github.com/ioggstream
+    $type: foaf:Person
+    schema:givenName: Roberto
+    schema:familyName: Polli
+
+  - $id: https://github.com/TallTed
+    $type: foaf:Person
+    schema:givenName: Ted
+    schema:familyName: Thibodeau
```

Comment:
I don't know whether this is better
```suggestion
    schema:familyName: Thibodeau Jr
```

Reply:
or this is better
```suggestion
    schema:familyName: Thibodeau
    schema:honorificSuffix: Jr
```

Reply:
[schema.org](https://schema.org/Person) doesn't mention Jr, Sr, III, etc., so I'm not sure where to put my `Jr`.